protocol/test/foundry/field/auction-math.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+from eth_abi import encode_single
+import math
+from decimal import Decimal
+
+def main(args):
+    if(args.type == "fracExp"):
+        fracExp(args.input_1, args.input_2)
+    elif(args.type == "morningAuctionLog"):
+        morningAuctionLog(args.input_1, args.input_2)
+    else:
+        raise(Exception("Unknown type"))
+
+def fracExp(beanReward, blocks):
+    # cap at 25 blocks
+    blocks = blocks if blocks < 25 else 25
+    newReward = (beanReward) * pow(1.01, blocks * 12) 
+    enc = encode_single('uint256', int(newReward))
+
+    # this print statement must be here for ffi to work
+    print("0x" + enc.hex())
+
+def morningAuctionLog(t, blocks): 
+    # https://github.com/BeanstalkFarms/Beanstalk/pull/133 
```

Comment:
Believe this is trying to highlight the formula implemented below, should note that's why the link is included, also ideally we can get a better permalink than this

Reply:
good point, updated